base/array.jl

```diff
@@ -974,13 +972,24 @@ end
 
 Insert the elements of `items` to the beginning of `a`.
 
+See also: [`unshift!`](@ref), [`append!`](@ref), [`insert!`](@ref)
+
 # Examples
 ```jldoctest
 julia> prepend!([3],[1,2])
 3-element Array{Int64,1}:
  1
  2
  3
+
+julia> prepend!([1, 2, 3], [4, 5, 6])
```

Comment:
What does this extra example show that the one just above doesn't?

Reply:
this is a remnant of earlier edits, will remove.